seed/challenges/02-data-visualization-certification/react-projects.json

```diff
@@ -202,6 +265,25 @@
       "tests": [],
       "type": "zipline",
       "challengeType": 3,
+      "nameRu": "Создайте Рогулайк Данжеон-Кроулер",
+      "descriptionRu": [
+        "<strong>Задание:</strong> Создайте приложение <a href='http://codepen.io' target='_blank'>CodePen.io</a>, функционал которого схож с этим: <a href='http://codepen.io/FreeCodeCamp/full/dGOOEJ/' target='_blank'>http://codepen.io/FreeCodeCamp/full/dGOOEJ/</a>.",
+        "<strong>Правило #1:</strong> Не подсматривайте в код приложения-примера. Разберитесь самостоятельно.",
+        "<strong>Правило #2:</strong> Приложение должно удовлетворять нижеприведённым <a href='http://en.wikipedia.org/wiki/User_story' target='_blank'>пользовательским историям</a>. Используйте любые библиотеки или API, которые потребуются. Придайте ему свой личный стиль.",
+        "<strong>Правило #3:</strong> Для создания этого проекта вы должны использовать Sass и React.",
+        "<strong>Пользовательская История:</strong> У меня есть жизни, уровень и оружие. Я могу подобрать лучшее оружие. Я могу подобрать очки здоровья.",
```

Comment:
`Я могу подобрать лучшее оружие` should be `Я могу подобрать оружие получше` because your translation means `I can pick up the best weapon` but it should mean `I can pick up a better weapon`.